modules/Bio/EnsEMBL/Variation/DBSQL/AnnotationFileAdaptor.pm

```diff
@@ -0,0 +1,280 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2018] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+
+=head1 CONTACT
+
+ Please email comments or questions to the public Ensembl
+ developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+ Questions may also be sent to the Ensembl help desk at
+ <http://www.ensembl.org/Help/Contact>.
+
+=cut
+
+#
+# Ensembl module for Bio::EnsEMBL::Variation::DBSQL::AnnotationFileAdaptor
+#
+#
+
+=head1 NAME
+
+Bio::EnsEMBL::DBSQL::AnnotationFileAdaptor
+
+=head1 SYNOPSIS
+  my $reg = 'Bio::EnsEMBL::Registry';
+
+  # set path to configuration file
+  # optionally it can be set as environment variable $ENSEMBL_VARIATION_ANNOTATION_CONFIG_FILE
+  $Bio::EnsEMBL::Variation::DBSQL::AnnotationFileAdaptor::CONFIG_FILE = '/path/to/annotation_config.json';
+
+  ## explicit use
+
+  # get Annotation File Adaptor
+  my $annotation_file_adaptor = $reg->get_adaptor('human', 'variation', 'AnnotationFile');
+  my $va  = $reg->get_adaptor('human', 'variation', 'variation');
+  my $v = $va->fetch_by_name('rs699');
+  my $vf = $v->get_all_VariationFeatures->[0];
+
+  my $gerp_file = $annotation_file_adaptor->fetch_by_type('gerp');
```

Comment:
Should be replaced by `fetch_by_annotation_type` in the description of this file and few others